compiler/rustc_parse/src/parser/path.rs

```diff
@@ -412,13 +420,16 @@ impl<'a> Parser<'a> {
     }
 
     /// Parses a single argument in the angle arguments `<...>` of a path segment.
-    fn parse_angle_arg(&mut self) -> PResult<'a, Option<AngleBracketedArg>> {
+    fn parse_angle_arg(&mut self, _ident_sp: Span) -> PResult<'a, Option<AngleBracketedArg>> {
```

Comment:
```suggestion
    fn parse_angle_arg(&mut self) -> PResult<'a, Option<AngleBracketedArg>> {
```
The parameter is no longer used.